users/tests/test_models.py

```diff
@@ -36,7 +36,7 @@ def test_user_creation(self):
     # Test that a default profile is automatically created after user registration
     def test_profile_creation(self):
 
-        test_profile = Profile.objects.get(user=User.objects.get(username="rswanson"))
+        test_profile = Profile.objects.get(user=User.objects.get(username=self.test_profile.user))
```

Comment:
I would expect this to be `test_profile = self.test_profile`

Reply:
Though I'm not sure what this is testing.  You're calling Profile.objects.create to create self.test_profile, so this seems to be testing Django's ability to create objects, which is a Django test, not a devroaast test.

Reply:
Don't remember what I was going for here. It was probably a mistake.